settings.gradle.kts

```diff
@@ -0,0 +1,7 @@
+
+rootProject.buildFileName = "gradle/project.gradle.kts"
+
+include(":widgets")
+val widgets = project(":widgets")
+widgets.projectDir = file("working dir/widgets")
```

Comment:
There is an additional `working dir/widgets/META-INF/main.kotlin_module` file that wasnt there before. Is this necessary? It is probably generated in the compilation process, but I do not remember this happening before (it could be Kotlin compiler update). Id rather not have it sit there , or anywhere for that matter.

Reply:
Sure, I'll see what it is

Reply:
It is a module file. Either java 9 module or Kotlin also adopted some sort of module system.

Reply:
I think the file is only generated if theres top level declaration in some of the widgets. Which should not be the case. Top lvl functions and extension functions in widgets are basically forbidden due to this.